packages/js/e2e-core-tests/specs/activate-and-setup/onboarding-tasklist.test.js

```diff
@@ -12,7 +12,7 @@ const {
 /**
  * External dependencies
  */
-const config = require( 'config' );
+const { config } = require( '@woocommerce/e2e-environment' );
```

Comment:
Up to this point we haven't had this dependency. We use `peerDependencies` among our e2e packages because they are located in the same folder.

Reply:
That might have been a headache! `peerDependencies` have been updated